pdns/recursordist/test-recursorcache_cc.cc

```diff
@@ -19,7 +19,7 @@ BOOST_AUTO_TEST_CASE(test_RecursorCacheSimple)
   std::vector<std::shared_ptr<DNSRecord>> authRecords;
   std::vector<std::shared_ptr<RRSIGRecordContent>> signatures;
   const DNSName authZone(".");
-  time_t now = time(nullptr);
+  time_t now = INT_MAX - 15;
```

Comment:
need to remember to remove this, right?

Reply:
well, my plan was to do both a check for time(nullptr); and `INT_MAX - 15`;